src/app/panel-one/page.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import useAbsDeaths from '../data/AbsDeaths';
+
+
+const useProcessedData = () => {
+    const absDeathsData = useAbsDeaths();
+    // console.log('absDeathsData Arr', absDeathsData);
+
+    const [globalDeathsByYear, setglobalDeathsByYear] = useState({})
+
+    useEffect(() => {
+        if (absDeathsData) {
+            let totalDeaths2019: number = Math.round(getTotalsByYear(absDeathsData, 2019, 'mean'));
+            let totalDeaths2018: number = Math.round(getTotalsByYear(absDeathsData, 2018, 'mean'));
+            let totalDeaths2017: number = Math.round(getTotalsByYear(absDeathsData, 2017, 'mean'));
+            let totalDeaths2016: number = Math.round(getTotalsByYear(absDeathsData, 2016, 'mean'));
+
+            let newGlobalDeathsByYear = {
+                '2016': totalDeaths2016,
+                '2017': totalDeaths2017,
+                '2018': totalDeaths2018,
+                '2019': totalDeaths2019,
+            };
+
+            setglobalDeathsByYear(newGlobalDeathsByYear);
+
+        }
+    }, [absDeathsData])
+
+    return globalDeathsByYear;
+}
+
+
+//create total # deaths / year object
+function getTotalsByYear (array: any, year: number, quartile: string) {
```

Comment:
Let's mark the parameter and return type for this functions.

Reply:
Updated the getTotalsByYear function with explicit types and added comments for clarity.